waiter/src/waiter/settings.clj

```diff
@@ -365,6 +365,8 @@
                                                 :default {:factory-fn 'waiter.authorization/noop-authorizer}}
                                    :cluster-name "waiter"
                                    :container-running-grace-secs 90
+                                   :custom-options {:reverse-proxy-flag "GRPC_TRANSCODER"
+                                                    :reverse-proxy-offset 1}
```

Comment:
We should not be using `custom-options`, these should be configured on the `proxy-options` instead.